scripts/test_review_gate.py

```
@@ -136,7 +136,39 @@ def test_prose_only_edit_is_minor(self):
-        self.assertIn(f"CLASS={gate.CLASS_MINOR}", self._classify(root))
+        output = self._classify(root)
+        self.assertIn(f"CLASS={gate.CLASS_MINOR}", output)
+        # `minor`はCodeRabbitの要否と紛れる文言ではないため、説明行を出さない。
+        self.assertNotIn("meaning:", output)
+
+    def test_review_required_explains_it_is_not_a_coderabbit_review(self):
+        """`CLASS=review-required`は`minor`経路を採れないという意味であり、
+        CodeRabbit reviewの要否ではない。**出力のどこにもその区別が無いと、
+        `review-required`という名前だけで読み違えられる。**
+
+        **`Change-Class`／`Self-Review`のtrailerは、値と無関係に無条件で
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `／` (FULLWIDTH SOLIDUS). Did you mean `/` (SOLIDUS)?

(RUF002)